recipes/libgraphene/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "graphene" %}
+{% set version = "1.10.4" %}
+{% set sha256 = "b2a45f230f332478553bd79666eca8df1d1c6dbf208c344ba9f5120592772414" %}
+
+package:
+    # graphene is already taken by an unrelated python package
+    name: "lib{{ name|lower }}"
+    version: {{ version }}
+
+source:
+    sha256: {{ sha256 }}
+    url: https://github.com/ebassi/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.xz
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
```

Do you want to skip Windows entirely for now rather than deploying a build which is missing the `gobject-introspection` bit?
```suggestion
  skip: true  # [win]
```

---

I think it's fine as is now. The Windows build works fine, as long as I don't build the gobject-introspection bindings. I will fix that once the feedstock is up and running.